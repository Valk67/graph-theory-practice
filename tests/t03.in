13 19
0 4
0 2
0 1
4 2
1 2
2 6
2 5
6 10
6 7
5 8
8 7
8 9
10 9
10 11
11 12
9 3
12 9 
12 3
8 1
