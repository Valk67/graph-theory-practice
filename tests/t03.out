Testing BFS on node 0.
0 4 2 1 6 5 10 7 8 9 11 3 12 
Testing print path on node 3.
0 2 6 10 9 3 
Testing print levels on node 0.
0 
4 2 1 
6 5 
10 7 8 
9 11 
3 12 
Testing DFS.
0 4 2 6 10 9 3 11 12 7 5 8 1 
Testing to check if a graph has a cycle.
Graph has a cycle.
